docker-compose.yml

```diff
@@ -46,6 +61,11 @@ services:
       - REDIS__HOSTNAME=redis
       - REDIS__PORT=6379
       - ELASTICSEARCH__URL=http://elasticsearch:9200
+      - MINIO__ENDPOINT=minio
+      - MINIO__PORT=9000
+      - MINIO__USE_SSL=false
+      - MINIO__ACCESS_KEY=ChangeMeAccess
+      - MINIO__SECRET_KEY=ChangeMeKey
```

Comment:
shouldn't be S3 instead of MINIO ?

Reply:
For the moment we are not sure of the compatibility with S3, the configuration was named "Minio".